diff --git a/CMakeLists.txt b/CMakeLists.txt
index 12273b3..442c19a 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -281,7 +281,7 @@ else()
             add_subdirectory(src/tbbmalloc_proxy)
         endif()
     endif()
-    if (NOT BUILD_SHARED_LIBS)
+    if (NOT BUILD_SHARED_LIBS OR SKBUILD)
         message(STATUS "TBBBind build targets are disabled due to unsupported environment")
     else()
         add_subdirectory(src/tbbbind)
@@ -297,7 +297,7 @@ else()
 
         install(EXPORT ${PROJECT_NAME}Targets
                 NAMESPACE TBB::
-                DESTINATION ${CMAKE_INSTALL_LIBDIR}/cmake/${PROJECT_NAME}
+                DESTINATION ${CMAKE_INSTALL_DATAROOTDIR}/cmake/${PROJECT_NAME}
                 COMPONENT devel)
         file(WRITE ${CMAKE_CURRENT_BINARY_DIR}/${PROJECT_NAME}Config.cmake
                    "include(\${CMAKE_CURRENT_LIST_DIR}/${PROJECT_NAME}Targets.cmake)\n")
@@ -311,7 +311,7 @@ else()
 
         install(FILES "${CMAKE_CURRENT_BINARY_DIR}/${PROJECT_NAME}Config.cmake"
                       "${CMAKE_CURRENT_BINARY_DIR}/${PROJECT_NAME}ConfigVersion.cmake"
-                DESTINATION ${CMAKE_INSTALL_LIBDIR}/cmake/${PROJECT_NAME}
+                DESTINATION ${CMAKE_INSTALL_DATAROOTDIR}/cmake/${PROJECT_NAME}
                 COMPONENT devel)
 
         install(FILES "README.md"
diff --git a/src/tbb/CMakeLists.txt b/src/tbb/CMakeLists.txt
index 8c84a0b..605e81d 100644
--- a/src/tbb/CMakeLists.txt
+++ b/src/tbb/CMakeLists.txt
@@ -87,13 +87,6 @@ target_compile_options(tbb
     ${TBB_COMMON_COMPILE_FLAGS}
 )
 
-# Avoid use of target_link_libraries here as it changes /DEF option to \DEF on Windows.
-set_target_properties(tbb PROPERTIES
-    DEFINE_SYMBOL ""
-    VERSION ${TBB_BINARY_VERSION}.${TBB_BINARY_MINOR_VERSION}
-    SOVERSION ${TBB_BINARY_VERSION}
-)
-
 tbb_handle_ipo(tbb)
 
 if (TBB_DEF_FILE_PREFIX) # If there's no prefix, assume we're using export directives
